react-app/src/components/ProposalDetailScreen/ProposalDetailScreen.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { Navigate, useParams } from "react-router-dom";
+import AppRoutes from "../../navigation/AppRoutes";
+import Paper from "../common/Paper/Paper";
+
+const ProposalDetailScreen: React.FC = () => {
+  const { id } = useParams();
+
+  if (!id) return <Navigate to={AppRoutes.Proposals} />;
+
+  return (
+    <>
+      <Paper>#{id} Proposal Summary Placeholder</Paper>
+      <Paper>Description Placeholder</Paper>
+      <Paper>Votes and Deposit Placeholder</Paper>
+      <Paper>Comments Placeholder</Paper>
+    </>
```

Comment:
We should not use fragments as the screen's main container as that would assume the parent component to have the specific set of styles for the screen to work. It might seem repetitive but we might want to use a div with flex here